plugins/reach/plugins/OpenCalais/batch/KReachVendorTaskOpenCalaisProcessorEngine.php

```diff
@@ -0,0 +1,777 @@
+<?php
+
+require_once (__DIR__ . DIRECTORY_SEPARATOR . 'Constants.php');
+require_once (__DIR__ . DIRECTORY_SEPARATOR . 'RuleEngine.php');
+
+/**
+ * @package plugins.openCalaisReachVendor
+ * @subpackage batch
+ */
+class KReachVendorTaskOpenCalaisProcessorEngine extends KReachVendorTaskProcessorEngine
```

Comment:
generic remark, all the files starting with kSomething, should be lowerCamelCase/
In this case kReachVendorTaskOpenCalaisProcessorEngine , and kReachVendorTaskProcessorEngine
Please rename file and class